<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>650</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convives Communication Room</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>C:/Users/Admin/Desktop/at_2.png</normaloff>C:/Users/Admin/Desktop/at_2.png</iconset>
  </property>
  <property name="toolTipDuration">
   <number>2</number>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>351</y>
      <width>621</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="opaqueResize">
     <bool>true</bool>
    </property>
    <property name="handleWidth">
     <number>3</number>
    </property>
    <widget class="QLineEdit" name="msgbox"/>
    <widget class="QPushButton" name="send">
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>621</width>
      <height>331</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="opaqueResize">
     <bool>true</bool>
    </property>
    <widget class="QPlainTextEdit" name="chat1">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="chat2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="cover">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="idLabel">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>141</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>ID</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="pwLabel">
     <property name="geometry">
      <rect>
       <x>185</x>
       <y>161</y>
       <width>50</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="idInput">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>140</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhSensitiveData</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="pwInput">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>160</y>
       <width>261</width>
       <height>20</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhExclusiveInputMask</set>
     </property>
    </widget>
    <widget class="QPushButton" name="loginButton">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>190</y>
       <width>78</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelLoginButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>190</y>
       <width>78</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </widget>
   <zorder>splitter_2</zorder>
   <zorder>splitter</zorder>
   <zorder>cover</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogin">
    <property name="title">
     <string>&amp;Session</string>
    </property>
    <addaction name="actionClear_chat"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menuLogin"/>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClear_chat">
   <property name="text">
    <string>&amp;Clear chat</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>idInput</tabstop>
  <tabstop>pwInput</tabstop>
  <tabstop>loginButton</tabstop>
  <tabstop>cancelLoginButton</tabstop>
  <tabstop>msgbox</tabstop>
  <tabstop>send</tabstop>
  <tabstop>chat1</tabstop>
  <tabstop>chat2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>msgbox</sender>
   <signal>returnPressed()</signal>
   <receiver>send</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>630</x>
     <y>391</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>send</sender>
   <signal>clicked()</signal>
   <receiver>chat2</receiver>
   <slot>paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>630</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pwInput</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>218</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>idInput</sender>
   <signal>returnPressed()</signal>
   <receiver>loginButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
